ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-722] Update cannot check dwp status screen

## Description of change
Update content on cannot check dwp status screen

## Link to relevant ticket
[CRIMAPP-722](https://dsdmoj.atlassian.net/browse/CRIMAPP-722)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
<img width="1106" alt="Screenshot 2024-04-15 at 11 53 57" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/7006a1e9-888f-4a9a-befc-82a547fa1293">

### After changes:

https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/84ea29a4-e598-4d30-8f26-4f73696be92b

## How to manually test the feature
Note you will need to set the env variable BC_USE_DEV_MOCK to false to simulate the dwp api being down
Test the following scenario with the feature flag turned on and then turned off

Start an application
Enter a nino
Select a benefit type
You should see the screen Sorry we cannot check your client's info with DWP
Click try again now and the dwp check should attempt again
Click save and continue and you should be routed to the screen Do you have evidence your client gets ____?
Go back and click save and come back later and you should be routed to the task list


[CRIMAPP-722]: https://dsdmoj.atlassian.net/browse/CRIMAPP-722?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
